tools/explorer/lib/parse_input.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Main module for DBO explorer."""
+import ast
+from termcolor import colored
+from typing import List
+
+from lib import model
+from lib.model import StandardField
+
+DEFAULT_MATCHED_TYPES_LIST_SIZE = 10
+
+def _InputFieldsManually(ontology) -> List[StandardField]:
+  """Method to take individual field inputs from user and return as a list.
+
+  Args:
+    ontology: An instance of OntologyWrapper class.
+
+  Returns:
+    A list of StandardField objects.
+  """
+  standard_field_list = []
+  field_number = input('Number of entity fields: ')
+  for i in range(1, int(field_number)+1):
+    field_name = input(f'Field name #{i}: ')
+    standard_field = model.StandardField(
+        standard_field_name=field_name,
+        namespace_name=''
+    )
+    if ontology.IsFieldValid(standard_field):
+      standard_field_list.append(standard_field)
+    else:
+      continue
+  return standard_field_list
+
+def _InputFieldsFromCSV(ontology) -> List[StandardField]:
+  """Method to take batch field inputs in the form of a comma separated list of
+  fields from user and return as a list
+
+  Args:
+    ontology: An instance of OntologyWrapper class.
+
+  Returns:
+    A list of StandardField objects
+  """
+  standard_field_list = []
+  raw_fields = input('Paste your fields here: ')
+  raw_field_list = raw_fields.replace(' ', '').split(',')
+  for field in raw_field_list:
+    standard_field = StandardField(
+        standard_field_name=field,
+        namespace_name=''
+    )
+    if ontology.IsFieldValid(standard_field):
+      standard_field_list.append(standard_field)
+  return standard_field_list
+
```

Comment:
combine these two into one:

`def _InputFieldsFromUser(ontology) ...`
you can ask their preference in it if it makes it easier to implement.

or you can ask outside of the function and combine and call one method.

you would have one loop

currently it is unnecessary duplicate code please refactor.